jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9AttachProvider.java

```diff
@@ -222,6 +226,7 @@ public String type() {
 		return "Java SE"; //$NON-NLS-1$
 	}
 
+	/*[IF JAVA_SPEC_VERSION < 24]*/
 	private static void checkAttachSecurity() {
```

Comment:
I think there are some AttachAPI security tests, do they pass with this change?

Reply:
My personal build build with sanity and extended personal tests looks okay. I don't think any of the failures are related. https://hyc-runtimes-jenkins.swg-devops.com/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/25989/